packages/verification-vue-component/src/components/Verification/items/ResultDetail.vue

```diff
@@ -1,43 +1,128 @@
 <template>
-    <div class="result-detail">
+    <div class="result-detail"
+         :class="[verificationResultClass, {expanded: showDetails}]">
         <div class="detail-header">
```

Comment:
For me it would make sense to make the whole detail-header clickable instead of only the small down chevron. But maybe ask stephan what's his opion is about this.

Reply:
Full header is now clickable